library/testing/PseudoUnitTest.h

```diff
@@ -108,6 +110,14 @@ class PseudoUnitTest
     this->log(success, label, message);
   }
 
+  std::string toString(const f3d::color_t& color)
+  {
+    const double* value = color.data();
+    std::stringstream ss;
+    ss << value[0] << "," << value[1] << "," << value[2];
+    return ss.str();
+  }
+
```

Comment:
@snoyer I had to declare this

Reply:
Not a big fan of using the libf3d in the testing framework but I dont see a clean way to make this work.

Reply:
would it be possible to implement `stream << color_t` somewhere else instead?

Reply:
> Also not a big fan of changing code for testing reason.

Agreed, I was thinking adding `<<` could be done in the test code but I haven't tried it.
Another solution could be to subclass `PseudoUnitTest` in this specific test to implement the missing `toString()` there?

Reply:
good point, that would reduce the reach

Reply:
how about doing this in `TestSDKOptionsIO.cxx`:
```cpp
#include <export.h>
#include <options.h>

#include <iostream>

std::ostream& operator<<(std::ostream& os, const f3d::color_t& color)
{
  os << color[0] << "," << color[1] << "," << color[2];
  return os;
}

#include "PseudoUnitTest.h"
```
?
not great to shove code in between the includes but maybe it's acceptable for test stuff?

Reply:
that works, its better imo.